### PR TITLE
ref(*): kubernetes v1.11 support

### PR DESCRIPTION
**Pull request by adamreese**

### CODE REVIEW

File: pkg/kube/client_test.go [hunk 1]
```diff
@@ -98,32 +93,14 @@ func newResponse(code int, obj runtime.Object) (*http.Response, error) {
...
-
 type testClient struct {
 	*Client
 	*cmdtesting.TestFactory
 }
 
 func newTestClient() *testClient {
 	tf := cmdtesting.NewTestFactory()
-	tf.Namespace = core.NamespaceDefault
+	// tf.Namespace = core.NamespaceDefault
```
Comment (bacongobbler) [inline pkg/kube/client_test.go]: delete this?

Comment (technosophos) [inline pkg/kube/client_test.go]: Remove?

**Review by technosophos — APPROVED** [review]

File: pkg/kube/client_test.go (hunk 1 above)
Comment (adamreese) [reply pkg/kube/client_test.go]: deleted

Comment (adamreese) [reply pkg/kube/client_test.go]: removed

**Review by bacongobbler — APPROVED** [review]

**Pull request merged**